piston

```diff
@@ -68,23 +90,24 @@ case "$SUBCOMMAND" in
             -it \
             --name $CONTAINER_NAME \
             -e PISTON_LOG_LEVEL=DEBUG \
-            -e PISTON_FLAKE_PATH=/piston/packages \
+            -e PISTON_FLAKE_PATH=/piston/src \
             -e PISTON_RUNTIME_SET=$runtime_set \
-            -v $PWD:/piston/packages \
-            -d "$IMAGE_NAME_DEV:$IMAGE_TAG"
-    ;;
+            -v "$SCRIPT_DIR":/piston/src \
+            -v $DEV_VOLUME_NAME:/nix \
+            -d "$IMAGE_NAME_DEV:$IMAGE_TAG" \
+            bash -c "cd /piston/src/api && yarn run dev"
+        ;;
 
     test)
-        build_base
         docker run \
             --rm \
             -it \
-            -e PISTON_FLAKE_PATH=/piston/packages \
-            -v $PWD:/piston/packages \
+            -e PISTON_FLAKE_PATH=/piston/src \
+            -v "$SCRIPT_DIR":/piston/src \
+            -v $DEV_VOLUME_NAME:/nix \
             --name piston_test_runner \
-            -it "$IMAGE_NAME_DEV:$IMAGE_TAG" \
-            piston-test $1
-
+            "$IMAGE_NAME_DEV:$IMAGE_TAG" \
+            bash -c "/piston/src/api/src/bin/test.js $1"
```

Comment:
I would much rather prefer we test in the production environment, but caching the `/nix` directory could be an option